src/msg/Message.h

```diff
@@ -237,7 +237,13 @@
 #define MSG_MGR_COMMAND_REPLY     0x70a
 
 // *** ceph-mgr <-> MON daemons ***
-#define MSG_MGR_UPDATE     0x70b
+#define MSG_MGR_UPDATE            0x70b
+
+// *** nvmeof mon -> gw daemons ***
+#define MSG_MNVMEOF_GW_MAP        0x70c
+
+// *** gw daemons -> nvmeof mon  ***
+#define MSG_MNVMEOF_GW_BEACON     0x70d
```

Comment:
```suggestion
#define MSG_MNVMEOF_GW_MAP        0x800

// *** gw daemons -> nvmeof mon  ***
#define MSG_MNVMEOF_GW_BEACON     0x801
```

Use `0x800` as a bit mask for nvmeofgw messages.

Reply:
@batrick 🖖, appreciation for your review; please take a look at commit 696893d